File: Day_2/question8.c
#include<stdio.h>
int main()
{
    int num, num2 = 0, r,n;

    printf("Enter the number to reverse:");

    scanf("%d", &num);

    n = num;
    
    while(num>0){

        r=num%10;
        num2 = num2*10 + r;
        num=num/10;
    }

    if(n==num2)
    {
    printf("The entered number is a Paindrome");
    }

    else
    {       
     printf("The entered number is not a Paindrome");
    }
    
    return 0;
}

File: Day_2/question8.py
#WAP to check whether a number is palindrome.
n = int(input("Enter number : "))
x = n
rev = 0

while x > 0:
    r = x % 10
    rev = rev * 10 + r
    x = x // 10

print("Reverse of given number is" , rev)

if rev == n:
    print(n , "is a Palindrome")
else:
    print(n, "is not a Palindrome")